### ISSUE TITLE
kube-proxy expects to load kernel modules in v1.8.x

### ISSUE THREAD

**Issue by dghubble** [kubernetes-incubator/bootkube#741]
```
# kube-proxy
time="2017-10-19T22:42:27Z" level=warning msg="Running modprobe ip_vs failed with message: ``, error: exec: \"modprobe\": executable file not found in $PATH"
time="2017-10-19T22:42:27Z" level=error msg="Could not get ipvs family information from the kernel. It is possible that ipvs is not enabled in your kernel. Native loadbalancing will not work until this is fixed."
```

Mostly benign for now, but should be fixed. The image contains `sbin/modprobe` (which can't be found?) and we do not currently load `/lib/modules` in kube-proxy.

**Comment by rphillips** [comment]
This will be fixed in the next 1.8 release (https://github.com/kubernetes/kubernetes/pull/54325)

**Comment by dghubble** [comment]
We also need to be sure to mount `/lib/modules` like we do for Calico and a few others that run their own modprobes. Lemme send a patch.

**Comment by dghubble** [comment]
Resolved with Kubernetes v1.8.3 and #742

```
W1108 22:11:13.321752       1 server.go:191] WARNING: all flags other than --config, --write-config-to, and --cleanup are deprecated. Please begin using a config file ASAP.
I1108 22:11:13.352515       1 server_others.go:122] Using iptables Proxier.
I1108 22:11:13.367019       1 server_others.go:157] Tearing down inactive rules.
I1108 22:11:13.397747       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_max' to 131072
I1108 22:11:13.397916       1 conntrack.go:52] Setting nf_conntrack_max to 131072
I1108 22:11:13.398294       1 conntrack.go:83] Setting conntrack hashsize to 32768
I1108 22:11:13.398452       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_established' to 86400
I1108 22:11:13.398521       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_close_wait' to 3600
I1108 22:11:13.399653       1 config.go:202] Starting service config controller
```

**Comment by bamb00** [comment]
I'm getting the same error from kube-proxy in k8s v1.9.5.

	# kubectl logs kube-proxy-9pgwq -n kube-system
	I0719 19:26:42.583788       1 feature_gate.go:190] feature gates: map[]
	time="2018-07-19T19:26:42Z" level=warning msg="Running modprobe ip_vs failed with message: `modprobe: ERROR: could not insert 'ip_vs': Exec format error\ninsmod /lib/modules/3.10.0-862.9.1.el7.x86_64/kernel/net/netfilter/ipvs/ip_vs.ko.xz`, error: exit status     1"
	time="2018-07-19T19:26:42Z" level=error msg="Could not get ipvs family information from the kernel. It is possible that ipvs is not enabled in your kernel. Native loadbalancing will not work until this is fixed."
	W0719 19:26:42.591863       1 server_others.go:289] Flag proxy-mode="" unknown, assuming iptables proxy
	I0719 19:26:42.592958       1 server_others.go:138] Using iptables Proxier.
	W0719 19:26:42.593341       1 server.go:586] Failed to retrieve node info: Get https://172.22.87.243:6443/api/v1/nodes/ccchan-master-01: dial tcp 172.22.87.243:6443: getsockopt: connection refused
	W0719 19:26:42.593420       1 proxier.go:463] invalid nodeIP, initializing kube-proxy with 127.0.0.1 as nodeIP
	W0719 19:26:42.593431       1 proxier.go:468] clusterCIDR not specified, unable to distinguish between internal and external traffic
	I0719 19:26:42.593498       1 server_others.go:171] Tearing down inactive rules.
	I0719 19:26:42.619173       1 server.go:426] Version: v1.9.5
	I0719 19:26:42.626708       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_max' to 524288
	I0719 19:26:42.626770       1 conntrack.go:52] Setting nf_conntrack_max to 524288
	I0719 19:26:42.626809       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_established' to 86400
	I0719 19:26:42.626823       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_close_wait' to 3600
	I0719 19:26:42.626978       1 config.go:102] Starting endpoints config controller
	I0719 19:26:42.626999       1 controller_utils.go:1019] Waiting for caches to sync for endpoints config controller
	I0719 19:26:42.627055       1 config.go:202] Starting service config controller
	I0719 19:26:42.627061       1 controller_utils.go:1019] Waiting for caches to sync for service config controller
	E0719 19:26:42.627819       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://172.22.87.243:6443/api/v1/endpoints?limit=500&resourceVersion=0: dial tcp 172    .22.87.243:6443: getsockopt: connection refused
	E0719 19:26:42.627879       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://172.22.87.243:6443/api/v1/services?limit=500&resourceVersion=0: dial tcp 172.22    .87.243:6443: getsockopt: connection refused
	E0719 19:26:42.627940       1 event.go:209] Unable to write event: 'Post https://172.22.87.243:6443/api/v1/namespaces/default/events: dial tcp 172.22.87.243:6443: getsockopt: connection refused' (may retry after sleeping)
	I0719 19:26:46.327165       1 controller_utils.go:1026] Caches are synced for endpoints config controller
	I0719 19:26:46.327331       1 controller_utils.go:1026] Caches are synced for service config controller
	E0719 19:26:46.355864       1 proxier.go:1667] Failed to execute iptables-restore: exit status 2 (iptables-restore v1.6.0: invalid mask `128' specified
	Error occurred at line: 15
	Try `iptables-restore -h' or 'iptables-restore --help' for more information.



	# kubectl version
	Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.5", GitCommit:"f01a2bf98249a4db383560443a59bed0c13575df", GitTreeState:"clean", BuildDate:"2018-03-19T15:59:24Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
	Server Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.5", GitCommit:"f01a2bf98249a4db383560443a59bed0c13575df", GitTreeState:"clean", BuildDate:"2018-03-19T15:50:45Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
